src/xdp-app-info-private.h

```diff
@@ -23,6 +23,12 @@
 
 #include "xdp-app-info.h"
 
+typedef enum
```

Comment:
`typedef enum _XdpAppInfoFlags`

tbh, I'm unsure why this even is a pattern in e.g. mutter but at least it makes it easy to grep for the declaration...

Reply:
It's nice because one can use ctags to go directly to the actual struct, instead of the typedef, by adding a _ in front of the name.